src/main/java/ladder/domain/LadderResult.java

```diff
@@ -16,45 +16,23 @@ private Map<Name, Result> result(Names names, Ladder ladder, Results results) {
         Map<Name, Result> ladderResult = new HashMap<>();
         for (int nameIdx = 0; nameIdx < names.size(); nameIdx++) {
             Name name = names.of(nameIdx);
-            int position = position(ladder, nameIdx);
-            Result result = results.of(position);
+            Position position = position(ladder, nameIdx);
```

Comment:
> 3단계 리뷰에 남겨주신 https://github.com/next-step/java-ladder/pull/1910#discussion_r1401322485의 경우, 값을 앞단에서 꺼내기 보다는 실제 사용하는 곳에서 꺼내쓰는게 좋다 생각하여 수정하지 않았습니다. 관련하여 제가 추가로 생각할 거리나 의견이 있으시다면 편하게 남겨주세요!

```suggestion
            Position 새로운관점 = ladder.blabla(nameIdx);
```

그렇다면 이렇게는 어때요? 값을 가진 객체에게 물어보는거죠!